### PR TITLE
Add additional install instructions + note about exiting Wash

### PR DESCRIPTION
**Pull request by ekinanp**
Signed-off-by: Enis Inan <enis.inan@puppet.com>

### CODE REVIEW

File: docs/installing_wash.md [hunk 1]
```diff
@@ -18,5 +18,8 @@ Wash is distributed as a single binary; the only prerequisite is `libfuse`. Here
 
 * Run `wash --verify-install` to ensure that the installation was successful
     * If anything fails, then check out the [known issues page]({{ '/known_issues' | relative_url }}) to see if the failure(s) correspond to any of the known issues. Otherwise, please don't hesitate to ask us on [slack](https://puppetcommunity.slack.com/app_redirect?channel=wash) for help!
+        * You can use `wash --verify-install` to test any fixes
```
Comment (MikaelSmith) [inline docs/installing_wash.md]: I don't think I'd indent this, putting at the same level as the previous note seems fine.

**Review by MikaelSmith — APPROVED** [review]

File: docs/installing_wash.md (hunk 1 above)
Comment (ekinanp) [reply docs/installing_wash.md]: Yeah, putting on the same level does look a bit better. Updated.

**Comment by ekinanp** [comment]
Updated again to add `curl` commands for downloading the wash tgz.

File: docs/installing_wash.md [hunk 2]
```diff
@@ -11,12 +11,24 @@ Wash is distributed as a single binary; the only prerequisite is `libfuse`. Here
...
+            curl -sfLO https://github.com/puppetlabs/wash/releases/download/{WASH_VERSION}/wash-{WASH_VERSION}-x86_64-unknown-linux.tgz
+            ```
+        * On MacOS:
+            ```
+            curl -sfLO https://github.com/puppetlabs/wash/releases/download/{WASH_VERSION}/wash-{WASH_VERSION}-x86_64-apple-darwin.tgz
+            ```
+
+      where `{WASH_VERSION}` is the [latest Wash version](https://github.com/puppetlabs/wash/releases/latest) (e.g. `0.16.0`). After downloading the `.tgz`, run the following commands:
         * `tar -xvzf <path_to_downloaded_wash_tgz>` (unpack it)
-        * `chmod +x <path_to_wash>` (ensure it's executable)
```
Comment (ekinanp) [inline docs/installing_wash.md]: I noticed that `tar -xvzf` places the extracted `wash` binary in the dir. that you ran the command, which is why I made this change.

**Review by MikaelSmith — APPROVED** [review]

**Pull request merged**